src/backend/tests/integration/components/mcp/test_mcp_component.py

```
@@ -1,11 +1,18 @@
+import pytest
+
 from tests.integration.utils import run_single_component
 
 
+# TODO: Add more tests for MCPToolsComponent
+@pytest.mark.asyncio
 async def test_mcp_component():
     from langflow.components.data.mcp_component import MCPToolsComponent
```

Pylint (3.3.7)

[error] No name 'components' in module 'langflow'

(E0611)